packages/gp2-components/src/organisms/UserProjects.tsx

```diff
@@ -0,0 +1,154 @@
+import { gp2 } from '@asap-hub/model';
+import {
+  Button,
+  chevronCircleDownIcon,
+  chevronCircleUpIcon,
+  lead,
+  Link,
+  Paragraph,
+  pixels,
+} from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { useState } from 'react';
+import { StatusPill } from '../molecules';
+import colors from '../templates/colors';
+
+const { rem, tabletScreen } = pixels;
+
+const contentStyles = css({
+  padding: `${rem(16)} 0`,
+});
+const rowStyles = css({
+  borderBottom: `1px solid ${colors.neutral500.rgb}`,
+  marginBottom: rem(12),
+  padding: `${rem(16)} 0 ${rem(12)}`,
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    display: 'flex',
+  },
+});
+const textStyles = css({
+  color: lead.rgb,
+});
+const hideStyles = css({
+  [`:nth-of-type(n+6)`]: { display: 'none' },
+});
+const headingTopStyles = css({
+  display: 'none',
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
```

Comment:
looks like this is not used much in gp2. Try using the media queries from 

https://github.com/yldio/asap-hub/blob/dace4dcfa6933c8a6b1063d6fbde1efb130b59ed/packages/gp2-components/src/layout.ts#L15-L16